app/vmselect/promql/simplify_test.go

```diff
@@ -0,0 +1,89 @@
+package promql
```

Comment:
I moved all the tests that involved simplification to this file (see comment above for reasoning)